pkg/buildcontext/file.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package buildcontext
+
+import (
+	"strings"
+
+	"github.com/GoogleContainerTools/kaniko/pkg/util"
+)
+
+// BuildContext unifies calls to download and unpack the build context.
+type File struct {
+}
+
+func (f *File) UnpackTarFromBuildContext(buildContext string, directory string) error {
+	srcContext := strings.TrimPrefix(buildContext, "file://")
```

Comment:
I think you changed this to `dir://`? Also, we should probably change the name of the struct to reflect that!

Reply:
Missed that piece there and I’ll go forward to rename the struct and file as well.